### ISSUE TITLE
Request : Add a CDN option

### ISSUE THREAD

**Issue by aussetg** [ruhoh/ruhoh.rb#51]
Would it be possible to ask Ruhoh to generate all links relative to a new config variable ( site_cdn ) so that all files except .html files are served from site_cdn ?

Thanks

**Comment by plusjade** [comment]
Yes that is possible and planned actually. I have some skeleton code that accounts for a cdn options. Mainly that's the reason for the variable paths, e.g.: {{paths.media}} though to be honest I still have to figure out all the edge cases and (hopefully) intuitive solution.
